dotCMS/src/integration-test/java/com/dotcms/publisher/business/PublishAuditAPITest.java

```diff
@@ -0,0 +1,173 @@
+package com.dotcms.publisher.business;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+
+import com.dotcms.datagen.UserDataGen;
+import com.dotcms.publisher.bundle.bean.Bundle;
+import com.dotcms.publisher.business.PublishAuditStatus.Status;
+import com.dotcms.util.IntegrationTestInitService;
+import com.dotmarketing.business.APILocator;
+import com.dotmarketing.exception.DotDataException;
+import com.dotmarketing.util.UUIDGenerator;
+import com.liferay.portal.model.User;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.List;
+import org.junit.AfterClass;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class PublishAuditAPITest {
+
+    private static PublishAuditAPI publishAuditAPI;
+
+    @BeforeClass
+    public static void prepare() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A method/constructor should not explicitly throw java.lang.Exception](https://app.codacy.com/app/dotCMS/core/pullRequest?prid=4502981)